### PR TITLE
Fixing Usage docs

### PR DESCRIPTION
**Pull request by mcbenjemaa**
used fresh clusterctl config. 
Got:

```
> clusterctl init --infrastructure proxmox --ipam in-cluster
Fetching providers
Error: failed to get provider components for the "in-cluster" provider: failed to read "ipam-components.yaml" from provider's repository "ipam-in-cluster": release not found for version download, please retry later or set "GOPROXY=off" to get the current stable release: 404 Not Found
```

*Issue #, if available:*

*Description of changes:*

*Testing performed:*

### CODE REVIEW

File: docs/Usage.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ We need to add the IPAM provider to your clusterctl config file `~/.cluster-api/
 ```yaml
 providers:
   - name: in-cluster
-    url: https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/releases/download/v0.1.0-alpha.3/ipam-components.yaml
+    url: https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/releases/v0.1.0-alpha.3/ipam-components.yaml
```
Comment (wikkyk) [inline docs/Usage.md:55]: I don't know how that's meant to work, the proposed url is a 404 for me.

Comment (mcbenjemaa) [reply docs/Usage.md:55]: Dont test the url. 

Test clusterctl

Comment (mcbenjemaa) [reply docs/Usage.md:55]: release not found for version download, plea

Comment (mcbenjemaa) [reply docs/Usage.md:55]: **release not found for version download**

**Pull request merged**